Python/DataAugmentationUtilsPackage/DataAugmentationUtils/DataAugmentation.py

```diff
@@ -11,20 +11,20 @@
 
 ################################# Augmentaiton Pipelines ###############################################
 
-def point_based_aug(out_dir, orig_img_list, orig_point_list, num_samples, num_PCA=0, sampler_type="KDE", mixture_num=0):
+def point_based_aug(out_dir, orig_img_list, orig_point_list, num_samples, num_PCA=0, sampler_type="kde", mixture_num=0):
```

Comment:
Also, I would use num_pca_modes rather than num_PCA (not consistent and not obvious).

Reply:
I made this num_dim everywhere so that it is consistent if we ever add another form of embedding.